src/v/kafka/client/test/fixture.h

```diff
@@ -74,4 +74,51 @@ class kafka_client_fixture : public redpanda_thread_fixture {
         add_topic(tp_ns, partitions).get();
         return tp_ns;
     }
+
```

Comment:
nit: not blocking at all, but it'd be really nice to put all the context from CLs into commits, especially for these one-commit PRs. It's a lot easier to inspect history from commits, rather than having to come back to the PRs.

Reply:
Ah I didn't make the CL in the cover letter. That's my bad.